data/accounts/store.go

```diff
@@ -46,6 +46,7 @@ func secureUser(user *User) *User {
 	return user
 }
 
+// nolint : dupl
 func (s *Store) getUser(ctx context.Context, name string) (user *User, err error) {
 	if user, err = s.rawUser(ctx, name); err != nil {
```

Comment:
Lines 50 - 57

```
func (s *Store) getUser(ctx context.Context, name string) (user *User, err error) {
	if user, err = s.rawUser(ctx, name); err != nil {
		return nil, err
	}
	if user == nil {
		return nil, UserNotFound
	}
	return user, nil
}
```

Converting `=` to `:=`

```
func (s *Store) getUser(ctx context.Context, name string) (*User, error) {
	if user, err := s.rawUser(ctx, name); err != nil {
		return nil, err
	}
	if user == nil {
		return nil, UserNotFound
	}
	return user, nil
}
```
I'm not particularly a huge fan (especially of naked returns), but named return values can be convenient. I just think within the very same files at least there ought to be consistency. The previous similar function on Line 30 (`func (s *Store) rawUser(ctx context.Context, name string) (*User, error) {`)  flagged as a duplicate doesn't use named return values.